Assets/Scripts/UnityServices/Lobbies/LobbyAPIInterface.cs

```diff
@@ -133,5 +133,10 @@ public async void SendHeartbeatPing(string lobbyId)
         {
             await LobbyService.Instance.SendHeartbeatPingAsync(lobbyId);
         }
+
+        public async Task<ILobbyEvents> SubscribeToLobby(string lobbyId, LobbyEventCallbacks eventCallbacks)
+        {
+            return await LobbyService.Instance.SubscribeToLobbyEventsAsync(lobbyId, eventCallbacks);
+        }

```

Comment:
Not this line, but GetLobby(..) is now unused. I'll leave it up to you if you want to remove it.

Reply:
Since this is an API interface, it could make sense to keep it? But honestly since we will likely never use it bc LobbyEvents are so much better in our use case I think we can just remove it